extension/debug/FabricDebugConfigurationProvider.ts

```diff
@@ -138,6 +138,10 @@ export abstract class FabricDebugConfigurationProvider implements vscode.DebugCo
             // configuration provider, for example a fabric:go configuration with the go debug configuration
             // provider. This results in errors and we need to just force it to use our configuration as-is.
             delete resolvedConfig.name;
+
+            // We need this in order to differentiate between debug events
+            resolvedConfig.debugEvent = 'contractDebugging';
```

Comment:
Can we make this a static variable please

Reply:
Done!